src/greeter/UserModel.cpp

```diff
@@ -54,6 +56,9 @@ namespace SDDM {
         const QString facesDir = mainConfig.Theme.FacesDir.get();
         const QString defaultFace = QStringLiteral("%1/.face.icon").arg(facesDir);
 
+#if HAVE_QTACCOUNTSSERVICE
+        asAccountsManager = new QtAccountsService::AccountsManager;
```

Comment:
Hmmmm. Perhaps I should try to un-markdown @tpgxyz's output first.

```
../src/greeter/UserModel.cpp:60:27: error: assigning to 'QtAccountsService::AccountsManager *' from incompatible type 'QtAccountsService::AccountsManager **'; dereference with *
```

(Yeah, try to wrap your paste with ```` ``` ```` next time.)